src/hooks/usePriceEstimation.ts

```diff
@@ -112,3 +112,43 @@ export function usePriceEstimationWithSlippage(params: SlippageParams): Result {
 
   return { priceEstimation, isPriceLoading }
 }
+
```

Comment:
I created a new hook for exactly this https://github.com/gnosis/dex-react/blob/a9dfb5a33854f9c797f5e2ff31ae77c580b99dd0/src/hooks/useOwlAmountInBaseTokenUnits.ts

Maybe they could be merged somehow?

Reply:
mine is more generic :robot: